### ISSUE TITLE
Automate release process

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#42]
## User story

1. As a developer
2. I want to automate the release process
3. So that I can streamline deployment, and reduce release time.

## Acceptance criteria

- [ ] Automated build processes are successfully configured and integrated with version control.
- [ ] Automated tests are executed as part of the build pipeline, and all tests pass before deployment.
- [ ] Deployments to staging, testing, and production environments are automated and consistently successful.
- [ ] Release notes are automatically generated and included with each release.
- [ ] A rollback mechanism is in place and tested to ensure the ability to revert to previous versions if necessary.

## Definition of done (DoD)

- [ ] Added only after week 5
- [ ] The same for all features
- [ ] Here goes the project specific part

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release
